.github/workflows/go.yml

```diff
@@ -430,11 +430,11 @@ jobs:
       - name: Install go
         uses: actions/setup-go@v4
         with:
-          go-version: '1.18'
+          go-version: '1.19'
           cache: true
           cache-dependency-path: go/go.sum
       - name: Install staticcheck
-        run: go install honnef.co/go/tools/cmd/staticcheck@v0.3.3
+        run: go install honnef.co/go/tools/cmd/staticcheck@v0.4.5
```

Comment:
Can we read staticcheck version from `.env`?

```diff
diff --git a/.github/workflows/go.yml b/.github/workflows/go.yml
index c31ad0b77..ba059817d 100644
--- a/.github/workflows/go.yml
+++ b/.github/workflows/go.yml
@@ -434,7 +434,10 @@ jobs:
           cache: true
           cache-dependency-path: go/go.sum
       - name: Install staticcheck
-        run: go install honnef.co/go/tools/cmd/staticcheck@v0.3.3
+        shell: bash
+        run: |
+          . .env
+          go install honnef.co/go/tools/cmd/staticcheck@${STATICCHECK}
       - name: Build
         shell: bash
         run: ci/scripts/go_build.sh $(pwd)
```

Reply:
sure, i'll update the file with this